data/blog/opentelemetry-docker.mdx

```diff
@@ -217,6 +248,10 @@ service:
       receivers: [otlp, docker_stats]
       processors: [resourcedetection, batch]
```

Comment:
The modification involves updating the pipeline configuration that includes docker_stats. Specifically, the change replaces `[resourcedetection, batch]` with `[resourcedetection/docker, batch]`. This adjustment is not applied to the other pipeline that lacks `docker_stats`.